shuup/notify/templates/notify/admin/script_item_editor.jinja

```diff
@@ -120,7 +122,7 @@
     .iframe-content-block {
         background: #fff;
         padding-top: 0px !important;
-        padding: 30px;
+        padding-left: 30px;
```

Comment:
this breaks everything
![image](https://user-images.githubusercontent.com/8770370/55190699-2120b980-517f-11e9-964b-7db2ca4d444e.png)